testing/btest/scripts/base/protocols/http/http-lower-case.zeek

```diff
@@ -0,0 +1,11 @@
+# This tests whether the HTTP analyzer handles HTTP keyword in lower case correctly or not.
+#
+# @TEST-EXEC: zeek -C -r $TRACES/http/http-lower-case.pcap
```

Comment:
Switch to bare mode, load %INPUT and `@load base/protocols/http` within the test?